backend/src/nodes/pytorch_nodes.py

```diff
@@ -280,7 +286,7 @@ def check_can_interp(self, model_a: OrderedDict, model_b: OrderedDict):
 
     def run(
         self, model_a: PyTorchModel, model_b: PyTorchModel, amount: int
-    ) -> Tuple[PyTorchModel, int, int]:
+    ) -> Tuple[Union[PyTorchModel, PyTorchFaceModel], int, int]:
```

Comment:
How can 2 non-face models make a face model?

Reply:
They can't. How else would I type this so it can support both though? You can still interpolate face models together afaik